trainer.py

```diff
@@ -74,5 +75,12 @@ def cli_main(
 
 # %%
 if __name__ == "__main__":
+    # tracemalloc.start()
     cli_main()
+
+    # snapshot = tracemalloc.take_snapshot()
```

Comment:
This commented out tracing code should be removed before merging.

Reply:
Done